src/ontology/fbbt-edit.obo

```diff
@@ -62827,7 +62831,7 @@ creation_date: 2008-11-20T07:54:36Z
 [Term]
 id: FBbt:00007319
 name: cardial cell
-def: "Contractile cell which forms the inner, contractile tube of the dorsal vessel." [FlyBase:FBrf0031012, FlyBase:FBrf0089570]
+def: "Cell that forms the wall of the dorsal vessel. They are arranged in two contralateral rows connected by ventral and dorsal adhesion zones, leaving a central lumen (Medioni et al., 2008)." [FlyBase:FBrf0031012, FlyBase:FBrf0089570, FlyBase:FBrf0205728]
 synonym: "cardiomyocyte" EXACT [FlyBase:FBrf0151925]
 is_a: FBbt:00100318 ! somatic cell
 relationship: develops_from FBbt:00001666 ! cardioblast
```

Comment:
We probably need a new stage-neutral term for 'dorsal vessel' that this and several other terms could have a part_of relationship to.